motorutil/steer.go

```diff
@@ -0,0 +1,276 @@
+// Copyright ©2016 The ev3go Authors. All rights reserved.
```

Comment:
it's 2017 now :wink:

Reply:
Yes, but the code it ripped from the znap example, which was written last year.